openpype/hosts/houdini/plugins/publish/collect_rop_frame_range.py

```diff
@@ -2,40 +2,94 @@
 """Collector plugin for frames data on ROP instances."""
 import hou  # noqa
 import pyblish.api
+from openpype.lib import BoolDef
 from openpype.hosts.houdini.api import lib
+from openpype.pipeline import OptionalPyblishPluginMixin
 
 
-class CollectRopFrameRange(pyblish.api.InstancePlugin):
+class CollectRopFrameRange(pyblish.api.InstancePlugin,
+                           OptionalPyblishPluginMixin):
```

Comment:
You are not using the Optional nature of this plug-in but only want the attribute defs.

I feel like this should instead inherit from `OpenPypePyblishPluginMixin`